Dockerfile

```diff
@@ -9,21 +9,14 @@ ARG CONTAINER_USERID
 # Configure our user
 RUN usermod -u $CONTAINER_USERID frontend
 
-ADD src/api/Gemfile /obs/src/api/Gemfile
-ADD src/api/Gemfile.lock /obs/src/api/Gemfile.lock
```

Comment:
If you remove this the bundle will never be updated. And there also is no way to update the bundle.

Reply:
This works like this: If the gem bundle is updated (the Gemfiles change) those `ADD`s make sure that the intermediate image will differ from the previous build and all the following steps need to be run again (on top of this changed image). This will refresh the bundle when building the image.

If you remove this step, the intermediate images will not change, so everything you do afterward will never be run again.

Reply:
Okay, I removed the two commits